lib/cocoapods/generate/installer.rb

```diff
@@ -287,7 +285,7 @@ def add_test_spec_schemes_to_app_scheme(installer, app_project)
 
             testable = Xcodeproj::XCScheme::TestAction::TestableReference.new(target)
             testable.buildable_references.each do |buildable|
-              buildable.xml_element.attributes['ReferencedContainer'] = 'container:Pods.xcodeproj'
+              buildable.xml_element.attributes['ReferencedContainer'] = "container:Pods/#{File.basename(target.project.path)}"
```

Comment:
this fixes a bug in which schemes were not shared when multi-xcodeproj was used.

Reply:
double checking - we confirmed this worked with single proj or no? I thought we did, and possibly determined that this has been broken for a while

Reply:
it does work for single proj I confirmed. I am not sure if it has been broken, possibly.

Reply:
Is the Pods/ prefix correct, given the sandbox root isnt nested inside the installation root?

Reply:
It is correct yes because we do create a Pods/ sandbox directory with cocoapods-generate.

It would probably be more correct though to use `sandbox` path?